Source/NetworkReachabilityManager.swift

```diff
@@ -0,0 +1,208 @@
+// NetworkReachabilityManager.swift
+//
+// Copyright (c) 2014–2016 Alamofire Software Foundation (http://alamofire.org/)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import Foundation
+import SystemConfiguration
+
+/// Notification posted when network reachability status changes. The notification `object` contains the update
+/// network reachability status as an `NSNumber` which will need to be converted.
+public let NetworkReachabilityStatusDidChangeNotification = "com.alamofire.network.reachability.status.did.change"
```

Comment:
If we use reverse domain name notation here wouldn't it make more sense to have the string be: `"com.alamofire.network.reachability.statusDidChange"` or if you must: `"com.alamofire.network.reachability.status.didChange"`

I'd actually rather suggest and follow Apple's guidelines here which make the string equal to the notification name. See [https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingIvarsAndTypes.html](url) (look under _Notifications and Exceptions_ )

All of Apple's notifications are made up this way as well.